client/src/components/web-extension/CreateWallet/RecoveryPhrasePage.jsx

```diff
@@ -39,18 +39,26 @@ const RecoveryPhrasePage = () => {
 			/>
 			<div className="cd_we_create-wallet-layout--body cd_we_create-keyphrase--box">
 				<ul className="cd_we_create-keyphrase--column">
-					{dropRight(sharesToMnemonicArrays(keyPhraseShares), TOTAL_KEYWORDS / 2)?.map((word, index) => (
-						<li className="cd_we_keyphrase--word" key={`left-${word}`}>
-							<CanvasText text={`${index + 1}. ${word}`} width="80" height="22" />
-						</li>
-					))}
+					{dropRight(new Array(TOTAL_KEYWORDS).fill(), TOTAL_KEYWORDS / 2)?.map((_, index) => {
+						const eleIndex = index + 1;
+
+						return (
+							<li className="cd_we_keyphrase--word" key={`left-${sharesToMnemonic(keyPhraseShares).split(' ')[eleIndex - 1]}}`}>
```

Comment:
Should we use index for key ? This for viewing only so index should be fine

Reply:
Hi @djhahe, I updated it and must use nanoid to unique this key and to avoid wrong render value